code-of-conduct.md

```diff
@@ -0,0 +1,74 @@
+# Contributor Covenant Code of Conduct
```

Comment:
Will discuss with the jdm-contrib group on adding this. I don't have strong feelings on it, but wouldn't mind adding it.